### PR TITLE
feat: add three labels (i18n, support, tech-debt)

### PR DESCRIPTION
**Pull request by Rexogamer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ draft            | This issue needs more work
 duplicate        | This issue or pull request already exists
 enhancement      | New feature or request
 fluff            | Minor app improvement / quality of life change
+i18n             | Issues regarding translations/localisation support
```
Comment (initmd) [inline README.md:55]:
```suggestion
i18n             | Issues regarding translations/localization support
```

File: labels/config.json [hunk 2]
```diff
@@ -133,21 +133,31 @@
...
       {
         "name": "a11y",
         "color": "0E8A16",
         "description": "Issues regarding accessibility problems"
       },
+      {
+        "name": "i18n",
+        "color": "5d0f99",
+        "description": "Issues regarding translations/localisation support"
+      },
```
[the remark this answers is not in the document]
Comment (Rexogamer) [reply labels/config.json:160]: Why?

File: README.md (hunk 1 above)
Comment (insertish) [reply README.md:55]: This will make the Queen very unhappy.

Comment (janderedev) [reply README.md:55]: why

Comment (initmd) [reply README.md:55]: the Queen :trollface:

Comment (initmd) [reply README.md:55]: > This will make the Queen very unhappy.
> 
> (on a serious note: everything is kept to British English)

Now we can do it

Comment (Rexogamer) [reply README.md:55]: This will make the King very unhappy :trollface:

**Pull request merged**